cmd/dmverity-vhd/main.go

```diff
@@ -72,16 +74,26 @@ func main() {
 
 func fetchImageLayers(ctx *cli.Context) (layers []v1.Layer, err error) {
 	image := ctx.String(imageFlag)
+	tarballPath := ctx.String(tarballFlag)
 	ref, err := name.ParseReference(image)
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to parse image reference: %s", image)
 	}
 
-	local := ctx.GlobalBool(sourceFlag)
+	dockerDaemon := ctx.GlobalBool(dockerFlag)
```

Comment:
Also, should it be an error to specify both the docker and tarball flags, since latter overrides the former?

Reply:
agreed, this is addressed in my latest commit [here](https://github.com/SethHollandsworth/hcsshim/blob/14e015a50a7e916a7639da153f76ebc68b991b3d/cmd/dmverity-vhd/main.go#L90)